zypper-migration

```diff
@@ -491,6 +520,8 @@ begin
       end
     end
 
+    remove_services_no_migration_path(filtered_services, options[:verbose])
```

Comment:
This is called for every product/service in the migration path. It should be called after the whole loop to avoid removing something which is covered by subsequent entries in the migration path.